packages/samples/package.json

```diff
@@ -19,14 +19,22 @@
     "cli"
   ],
   "type": "module",
+  "exports": {
+    ".": {
+      "default": "./dist/src/index.js",
+      "types": "./dist/src/index.d.ts"
+    }
+  },
   "engines": {
     "node": ">=16.0.0"
   },
   "scripts": {
     "clean": "rimraf ./dist ./temp",
     "build": "tsc -p .",
     "watch": "tsc -p . --watch",
-    "regen-samples": "node scripts/regen-samples.js"
+    "test": "mocha",
```

Comment:
So, essentially, rush-test includes running sample tests (regenerating samples without replacing output)

How do you imagine we might regenerate particular samples (which I think was part of the original issue)?

Reply:
For example, could we set environment variables for CLEAN_SAMPLES and RECORD and then run the individual tests?

Reply:
First yes you need to set `RECORD=true` env variable. The `CLEAN_SAMPLES` one should not be used as it will delete ALL the samples and is only meant for a full regen.

THen you can run the sample name with `-g option` so I think `npm run test -g "my-test"`

I would add a script that set the RECORD=true for you so its easier to do on windows and then you can have `npm run test:regen -g "my-test"`

Reply:
But It hink the main issue was able to diagnose a single sample which this does well more than regenerating a single one.